packages/flutter/lib/src/material/theme_data.dart

```diff
@@ -483,6 +493,15 @@ class ThemeData extends Diagnosticable {
   /// Defaults to the current platform.
   final TargetPlatform platform;
 
+  /// An adaptiveness theme that describes which Material widget use a
+  /// Cupertino equivalent widget on iOS when supported.
+  ///
+  /// Defaults to do no automatic widget adaptations on any widget.
```

Comment:
why? material says switch should adapt, no?

Reply:
It just says 'may be switched'. I can switch them by default and override them to false in the gallery. Though that's an API breaking change.